src/pages/guideline/ios/1-01-foundation.md

```diff
@@ -0,0 +1,27 @@
+---
+templateKey: ios-guideline
+title: Foundation
+date: 2019-01-11T00:15:32.894Z
+featuredImage: "/img/control-button.png"
+description: >-
+  SAP Fiori provides a consistent and holistic user experience for SAP software. By creating visually pleasing designs with a strong focus on ease of use, the experience is intuitive and simple, across all devices. With effortless interaction patterns, the SAP Fiori UX is designed for a powerful impact across your enterprise.
+tags:
+  - Foundation
+  - SAP Fiori
+version: '1.01'
+tabs:
```

Comment:
tabs needs to be defined in https://github.com/devinea/mvx19-cms/blob/master/static/admin/config.yml#L77 
so that CMS can edit it. 
However we don;'t want to maintain the description in every page. 
I think we should have a field here like 
category: "Controls"

in config.yml
      - {label: "Category", name: "category", widget: "select", options: ["Controls", "Views", "Patterns","Floorplans"]}